views/extras.tmpl

```diff
@@ -46,9 +46,7 @@
 <header>
 <div class="top-nav">
     <div class="container">
-        <div class="logo">
-            <a class="no-underline" href="/"><img src="/images/logo-2018.svg" alt="dcr-icon"></a>
-        </div>
+        <a class="no-underline logo" href="/"><img src="/images/logo-2018.svg" alt="dcr-icon"></a>
```

Comment:
Some of this css especially in the nexthome is heavily chained together. Changing the order without changing the css used will affect the items alignment. In this case this css will no longer work..
```css
.container .logo a img {
        width: 70px !important;
    }
```
And there are many more just like this.

Reply:
Well, the CSS does need some work. I found this and similar styles in the code I built off from. I'm suggesting doing CSS cleanup in a separate PR